pkg/client-sdk/types.go

```diff
@@ -19,13 +20,16 @@ var (
 		GrpcClient: grpcclient.NewClient,
 		RestClient: restclient.NewClient,
 	}
+
+	liquidRegTestUrl  = getEnv("LIQUID_REGTEST_URL", "http://localhost:3001")
+	bitcoinRegTestUrl = getEnv("BITCOIN_REGTEST_URL", "http://localhost:3000")
```

Comment:
You can use EnvVars of `cli` whne creating flags

Reply:
yeah, but var supportedNetworks is [inside](https://github.com/sekulicd/ark/blob/52b21c5e687111250b365e89438503e2327ce107/pkg/client-sdk/types.go#L26) sdk, imo this is simplest way to inject diff values for regtest without touching existing arksdk interface, otherwise we would need to pass network and this is not good since network comes from ASP via GetInfo. Instead of passing it via CLI env var i decided to pass it throuugh regtest dockerfile's.

Reply:
Also..we used to have an --explorer flag!!

Look in the current doc https://arkdev.info/docs/quick-start/client/#setting-up-alice-and-bobs-clients

Reply:
This is for sure wrong on the SDK side then. Let's fix it!

The explorer URL SHOULD NOT be mandated by the connected ASP, as is used to fetch blockchain data (I guess to check onchain balance? Compute timestamps from blocks?). 

It's very likely user of the SDK wants to use their explorer, instead being us forcing them with some well-known explorer URLs.